tidy3d/components/eme/monitor.py

```diff
@@ -59,14 +71,23 @@ class EMEMonitor(AbstractMonitor, ABC):
 
     @abstractmethod
     def storage_size(
-        self, num_cells: int, num_eme_cells: int, num_freqs: int, num_modes: int
+        self,
+        num_cells: int,
+        num_transverse_cells: int,
+        num_eme_cells: int,
```

Comment:
the original `Monitor.storage_size` has a different call signature right? I wonder if we should add a generic `**kwargs` to the call signature there so that technically one could call the original Monitor.storage_size with these extra kwargs (If that situation arises) and not get errors.

Reply:
I'm not sure, it might be better to get an error if you pass something like `num_eme_cells` to a monitor which is not an `EMEMonitor`. Likewise for all these other parameters. And `tmesh` doesn't make sense as a parameter for an `EMEMonitor`. So I think it's better to have different function signatures.